### DISCUSSION TITLE
1.8.3

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#999] [Обновления]
## Список изменений

- Исправлена ошибка из-за которой кнопка могла не появляться без переустановки расширения (#995, #997)
- Исправлена работа слайдеров громкости в меню расширения (#996)
- Исправлено ускорение видео при зажатие на меню расширения

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.8.3'>1.8.3</a>.</em>

**Comment by Flamb3rg** [comment]
Начиная с версии 1.8.0 возникает ошибка Failed to request create session. Переустановка скрипта не помогает.
Перевод работает на версии 1.7.2

P.S. Огромное спасибо за это полезное расширение!

**Comment by warlockblack** [comment]
Кнопка на видео (в ютюбе) по-прежнему не появляется. Переустановка не помогла. 
Последняя версия расширения, которая работает с Safari 15.6.1 – это версия 1.5.3.1.

**Reply by ilyhalight** [reply]
нет возможности нормально проверить с сафари
